lib/src/accordion/accordion.component.scss

```diff
@@ -0,0 +1,88 @@
+$accordion-border: 1px solid #DDDDDD;
+$accordion-background-color: #F0F0F0;
```

Comment:
Should these colors be in the `colors.scss` file for reuse? Conversely is there some way these non-standard colors could be achieved with the `tint` and `shade` functions @corykon wrote?

Reply:
Unless the colors are used in other components I would say no. No reason to pollute the main color.scss